client/app/scripts/superdesk/editor/editor.js

```diff
@@ -271,28 +389,51 @@ angular.module('superdesk.editor', [])
             templateUrl: 'scripts/superdesk/editor/views/editor.html',
             link: function(scope, elem, attrs, ngModel) {
 
-                var editorElem;
+                var editorElem, spelling = false, timeout;
```

Comment:
But this is javascript, right, we should be able to read javascript if doing code reviews...